app/views/items/_adaptive_lend_button.html.erb

```diff
@@ -1,6 +1,6 @@
 
-<% if !current_user.nil? && current_user.id != item.owner %>
-    <% if item.lend_status == "pending_pickup" && !LendRequestNotification.find_by(receiver: item.owner, item: item, borrower: current_user).nil?  %>
+<% if !current_user.nil? && current_user != item.owning_user %>
+    <% if item.lend_status == "pending_pickup" && !LendRequestNotification.find_by(receiver: item.owning_user, item: item, borrower: current_user).nil?  %>
```

Comment:
We intentionally did not add the new sets at any place in the code since this PR should only provide the data model. We've tried to keep the changes to existing features as small as possible while maintaining the current features. That means that all features are not capable of groups as item owners yet and we therefore did not want to change it here.

If we would have implemented group functionality for some but not all features, the features without groups functionality would break. And adapting group functionality for all features would definitely be out of scope for this PR.